mix.exs

```diff
@@ -56,7 +56,7 @@ defmodule Xebow.MixProject do
 
       # Dependencies for specific targets
       {:xebow_rpi0,
-       github: "doughsay/xebow_rpi0", ref: "v1.10.2+xebow.2", runtime: false, targets: :xebow_rpi0}
+       github: "doughsay/xebow_rpi0", ref: "v1.10.2+xebow.2", runtime: false, targets: :keybow}
```

Comment:
@doughsay I recommend prefixing the Nerves system repo with `nerves_system_` to follow the convention that the other [Nerves targets](https://hexdocs.pm/nerves/targets.html#content) do, if you think that's a good idea.

Reply:
You're definitely right, this makes much more sense.  I should rename the system repo to be `nerves_system_kebow` too.